RELEASE_NOTES.md

```diff
@@ -1,16 +1,22 @@
-# Cosmos SDK v0.42.0 "Stargate" Release Notes
+# Cosmos SDK v0.42.2 "Stargate" Release Notes
 
-This release includes an important security fix for all non "Cosmos Hub" chains (e.g. any chain that does not use the default `cosmos` bech32 prefix), and a few performance improvements.
+This maintenance release includes various bugfixes and performance improvements, and it does not introduce any breaking changes.
 
-See the [Cosmos SDK v0.42.0 milestone](https://github.com/cosmos/cosmos-sdk/milestone/42?closed=1) on our issue tracker for further details.
+See the [Cosmos SDK v0.42.2 milestone](https://github.com/cosmos/cosmos-sdk/milestone/41?closed=1) on our issue tracker for further details.
 
-# Security fix: validator address conversion in evidence handling
+### Keyring UX improvement
 
-The security fix resolves the issue regarding incorrect handling of validators' consensus addresses. Because of this incorrect handling, Cosmos SDK apps that were not using the default `cosmos` Bech32 address prefix were not able to jail validators that committed misbehaviors such as double signing.
+A number of macOS [users have reported](https://github.com/cosmos/cosmos-sdk/issues/8809) that their operating system's `keychain` prompt them for password to unlock the
```

Comment:
is this just weird spacing on this PR or is there an unintended line break here?

Reply:
It's markdown, so the text will be rendered in a single line anyway